### PR TITLE
Redesign DataPermissions

### PR DESCRIPTION
**Pull request by vdnj**
Capture d’écran 2022-09-13 à 16 34 54

### CODE REVIEW

File: src/locales/fr.json [hunk 1]
```diff
@@ -430,7 +430,8 @@
     "numberOfApplications" : "%{smart_count} application |||| %{smart_count} applications",
     "applications": "Applications",
     "data": "Données",
-    "open": "Ouvrir"
+    "open": "Ouvrir",
+    "access": "Les applications disposant de cette permission peuvent accéder à votre/vos "
```
Comment (trollepierre) [inline src/locales/fr.json:434]: we need to use plural form using smart count (like line 430)

File: src/components/Permissions/DataPermissions.jsx [hunk 2]
```diff
@@ -1,35 +1,40 @@
...
+import CozyClient, {
+  Q,
+  useQuery,
+  isQueryLoading,
+  hasQueryBeenLoaded
+} from 'cozy-client'
 import { APPS_DOCTYPE, KONNECTORS_DOCTYPE } from 'doctypes'
-import { Link, useParams } from 'react-router-dom'
-import withAllLocales from 'lib/withAllLocales'
+import { completePermission } from './DataListHelpers'
```
Comment (trollepierre) [inline src/components/Permissions/DataPermissions.jsx]: if it's used on DataPermissions, this is not a DataListHelpers

Comment (trollepierre) [reply src/components/Permissions/DataPermissions.jsx]: let's move this method inside a helper file in Permissions/helpers.js

File: src/components/Permissions/DataPermissions.spec.jsx [hunk 3]
```diff
@@ -1,139 +0,0 @@
-import { render } from '@testing-library/react'
```
Comment (trollepierre) [inline src/components/Permissions/DataPermissions.spec.jsx:1]: let's re-add this test

File: src/locales/fr.json (hunk 1 above)
Comment (vdnj) [reply src/locales/fr.json:434]: I think smart_count counts the number of applications. This is not the same usage here. What I can try to do is checking if the last letter of the Permission is an "s". If this is an "s" then I write "vos" else I write "votre".

Comment (trollepierre) [reply src/locales/fr.json:434]: if our PM agrees to use votre/vos, let's keep it

**Review by trollepierre — APPROVED** [review]

**Pull request merged**